### ISSUE TITLE
the y in the input must be a value?can it be a list?for example,[1,0,1,0,1]

### ISSUE THREAD

**Issue by 1397201241** [facebook/prophet#1869]

**Comment by bletham** [comment]
No, it must be a single value, joint forecasting isn't possible. There is some discussion of this in #49.

**Comment by 1397201241** [comment]
get it,thanks!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "facebook/prophet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月21日(星期三) 上午9:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [facebook/prophet] the y in the input must be a value?can it be a list?for example,[1,0,1,0,1] (#1869)





 
No, it must be a single value, joint forecasting isn't possible. There is some discussion of this in #49.
